django_u2f/u2f_impl.py

```diff
@@ -0,0 +1,6 @@
+try:
+    from u2flib_server import u2f_v2 as u2f
+    U2F_ENABLED = True
+except ImportError:
+    U2F_ENABLED = False
+    u2f = None
```

Comment:
Bike shedding: Why two variables? Why not `if u2f is None` and `if u2f is not None`?